README.md

```diff
@@ -107,24 +111,32 @@ Follow these steps:
 
 This determines the owner of the credentials that are issued. You can either reuse an existing DID already configured, or you can generate a new identity running:`make generate-issuer-did` (or `generate-issuer-did-arm`) and a new issuer did must be in the environment variable `ISSUER_API_UI_ISSUER_DID` in `.env-api`
 
-
 ### Advanced setup
 
 Any variable defined in the config file can be overwritten using environment variables. The binding for this environment variables is defined in the function `bindEnv()` in the file `internal/config/config.go`
 
 An _experimental_ helper command is provided via `make config` to allow an interactive generation of the config file, but this requires Go 1.19.
 
-## Development
+## Development (UI)
 
-[TODO]
+Completing either option of the [installation](#installation) process yields the UI as a minified Javascript app. Any changes to the UI source code would necessitate a Docker image removal and re-build to apply them. In most development scenarios this is undesirable, so the UI app can also be run in development mode like any [React](https://17.reactjs.org/) application to enable hot module replacement ([HMR](https://webpack.js.org/guides/hot-module-replacement/)).
 
-- Developing the issuer API
-- Developing the UI API
-- Developing the UI (HMR enabled with env config)
+1. Make sure that the UI API is set up and running properly (default <http://localhost:3002>).
+2. Go to the `ui/` folder.
+3. Copy the `.env.sample` file as `.env`
+4. All variables are required to be set, with the exception of `VITE_ISSUER_LOGO`. The following are the corresponding variables present in the parent folder's `.env-api`, which need to be the same. Only `VITE_ISSUER_NAME` can differ for the UI to function in development mode.
+    - `VITE_API_URL -> ISSUER_API_UI_SERVER_URL`
+    - `VITE_API_USERNAME -> ISSUER_API_UI_AUTH_USER`
+    - `VITE_API_PASSWORD -> ISSUER_API_UI_AUTH_PASSWORD`
+    - `VITE_ISSUER_DID -> ISSUER_API_UI_ISSUER_DID`
+    - `VITE_ISSUER_NAME -> ISSUER_API_UI_ISSUER_NAME`
+    - `VITE_ISSUER_LOGO -> ISSUER_API_UI_ISSUER_LOGO`
+5. Run `npm start`
```

Comment:
`npm i` first

Reply:
I hadn't finished BTW, hence this PR being in draft :)

Reply:
Yep I know 🤣